demisto_sdk/commands/common/hook_validations/base_validator.py

```diff
@@ -130,17 +129,6 @@ def _load_conf_file(self):
         with open(self.CONF_PATH) as data_file:
             return json.load(data_file)
 
-    def are_tests_configured(self) -> bool:
-        """
-        Checks if a file (playbook or integration) has a TestPlaybook and if the TestPlaybook is configured in conf.json
-        And prints an error message accordingly
-        """
-        file_type = self.structure_validator.scheme_name
-        tests = self.current_file.get('tests', [])
-        if not self.yml_has_test_key(tests, file_type):
-            return False
-        return self.tests_registered_in_conf_json_file(tests)
-
     def tests_registered_in_conf_json_file(self, test_playbooks: list) -> bool:
```

Comment:
improve function name:
```suggestion
    def are_tests_registered_in_conf_json_file(self, test_playbooks: list) -> bool:
```